# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/native-diet.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dietlibc-0.33/Makefile.vanilla	2016-01-07 18:21:14.335717749 +0000
+++ dietlibc-0.33/Makefile	2016-01-07 18:22:08.051718757 +0000
@@ -63,10 +63,14 @@
 ifeq ($(MYARCH),armeb)
 ARCH=arm
 else
+ifeq ($(MYARCH),native)
+ARCH=native
+else
 $(error unknown architecture, please fix Makefile)
 endif
 endif
 endif
+endif
 endif
 endif
 endif
